src/Internal/Plutip/Spawn.purs

```diff
@@ -88,3 +84,20 @@ killOnExit child = do
     alive <- Ref.read aliveRef
     when alive do
       kill SIGINT child
+
+foreign import _rmdirSync :: FilePath -> Effect Unit
+
+-- foreign import _removeDirs :: FilePath -> Effect Unit
+
+killOnExitAndRemDir :: ChildProcess -> FilePath -> FilePath -> Effect Unit
+killOnExitAndRemDir child workingDir testClusterDir = do
+  aliveRef <- Ref.new true
+  ChildProcess.onExit child \_ -> do
+    Ref.write false aliveRef
+    _rmdirSync workingDir
+  Process.onExit \_ -> do
+    alive <- Ref.read aliveRef
+    _rmdirSync testClusterDir
+    _rmdirSync workingDir
+    when alive do
```

Comment:
This is not needed. It will be killed anyway.

Reply:
but that's all that `killOnExit` (used 4 times in `Plutip.Server`) does. does that mean that we should remove all those effects?